packages/node/test/unit/channels.spec.ts

```diff
@@ -0,0 +1,128 @@
+import * as cf from "@counterfactual/cf.js";
+
+import { StateChannelInfo } from "../../src/channel";
+import { Node } from "../../src/node";
+
+describe("Basic channel operations and integrity on the Node", () => {
+  let node: Node;
+  let result: boolean;
+  let error: string;
+  const channelAB = new StateChannelInfo(
+    cf.legacy.constants.B_ADDRESS,
+    cf.legacy.constants.A_ADDRESS,
+    cf.legacy.constants.C_ADDRESS,
+    {},
+    cf.legacy.constants.EMPTY_FREE_BALANCE
+  );
+
+  beforeEach(() => {
+    node = new Node({});
+  });
```

Comment:
with this small of a beforeEach you might as well just write the one line in each it clause